samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -136,12 +138,8 @@ static int init_led_device(void)
 		return -ENODEV;
 	}
 
-	ret = gpio_pin_configure(led_dev, LED_GPIO_PIN, GPIO_DIR_OUT);
-	if (ret) {
-		return ret;
-	}
-
-	ret = gpio_pin_write(led_dev, LED_GPIO_PIN, 0);
+	ret = gpio_pin_configure(led_dev, LED_GPIO_PIN, LED_GPIO_FLAGS |
+							GPIO_OUTPUT_INACTIVE);
```

Comment:
As a style thing (which we do not define) I find it easier to understand when split at the comma rather than within the argument expression:
```
ret = gpio_pin_configure(led_dev, LED_GPIO_PIN,
                         LED_GPIO_FLAGS | GPIO_OUTPUT_INACTIVE);
```

Reply:
I actually prefer the style where things are split to max line length and then at the bitwise or logical operator boundary myself.

Reply:
OK then.